### PR TITLE
Leaderboard standalone script

### PR DESCRIPTION
**Pull request by HxxxxxS**
Shit, didn't mean to close the other PR.

Here's a squashed commit of just the standalone script, without touching the main script. Updated to use new position data from API, and to check for new data every 30 sec.

### CODE REVIEW

File: leaderboard.user.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
+// ==UserScript==
+// @name         Monster Minigame Leaderboard
+// @namespace    https://github.com/hansskogvold/steamSummerMinigame
```
Comment (xPaw) [inline leaderboard.user.js]: needs to be SteamDatabase, and in update urls too

Comment (HxxxxxS) [inline leaderboard.user.js]: But there's nothing at those URLs atm. Should I still set it to that?

Comment (xPaw) [inline leaderboard.user.js]: Yes.

**Comment by iakat** [comment]
The script seems to make HTTP requests, but the data is not filled in the leaderboard, could you troubleshoot this?

**Comment by HxxxxxS** [comment]
weird. it's working for me. are you using chrome or firefox?

man. 15. jun. 2015 kl. 20.48 skrev Simone notifications@github.com:

**Comment by iakat** [comment]
I'm using Firefox.

**Pull request closed without merging**